bucket/webcopy.json

```
@@ -0,0 +1,58 @@
+{
+    "version": "1.8.0.652",
+    "description": "Copy websites locally for offline browsing",
+    "homepage": "https://www.cyotek.com/cyotek-webcopy",
+    "license": {
+        "identifier": "Unknown",
+        "url": "https://www.cyotek.com/cyotek-webcopy/license-agreement"
+    },
+    "url": "https://www.cyotek.com/downloads/get/setup-cyowcopy-1.8.0-build-652.exe",
+    "hash": "ab328d7823f66bd99038be854de76c2e3abac5684190852982d6aa4a3c46edaa",
+    "innosetup": true,
+    "bin": "wcopycmd.cmd",
+    "architecture": {
+        "32bit": {
+            "shortcuts": [
+                [
+                    "cyowcopy.exe",
+                    "WebCopy "
+                ],
+                [
+                    "lnkchkgui.exe",
+                    "Link Checker"
+                ],
+                [
+                    "uritest.exe",
+                    "URI Tester"
+                ]
+            ]
+        },
+        "64bit": {
+            "shortcuts": [
+                [
+                    "cyowcopy-x64.exe",
+                    "WebCopy "
+                ],
+                [
+                    "lnkchkgui-x64.exe",
+                    "Link Checker"
+                ],
+                [
+                    "uritest.exe",
+                    "URI Tester"
+                ]
+            ]
+        }
+    },
+    "checkver": {
+        "url": "https://www.cyotek.com/cyotek-webcopy/downloads",
+        "regex": "<a href=\"/cyotek-webcopy/revision-history#[-\\d]+\">([\\d.]+)</a>"
```

Why do you not scrape the revision history directly?

```suggestion
        "url": "https://www.cyotek.com/cyotek-webcopy/revision-history",
        "regex": "Version\\s*([\\d.]+)"
```